### PR TITLE
Add python-pypi-mirror

### PR DESCRIPTION
**Pull request by ethanwhite**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged. _No static libraries_
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ethanwhite** [comment]
@conda-forge/help-python, ready for review!

I am happy to be the maintainer of this package and am the only person listed as a maintainer.

File: recipes/python-pypi-mirror/meta.yaml
Comment (BastianZim) [inline recipes/python-pypi-mirror/meta.yaml:33]: Why is this only required for the tests?

File: recipes/python-pypi-mirror/LICENSE
Comment (BastianZim) [inline recipes/python-pypi-mirror/LICENSE:6]: As upstream is active, maybe you can ask them to include the license in the source distribution. But nothing blocking.

**Review by BastianZim — CHANGES_REQUESTED** [review]
One small question and the other thing is not a blocker. Then this is ready to go.

File: recipes/python-pypi-mirror/meta.yaml
Comment (ethanwhite) [reply recipes/python-pypi-mirror/meta.yaml]: There was a new release since I submitted this removed the git requirement entirely.

**Comment by ethanwhite** [comment]
A new release has come out since I submitted this PR that removes the `git` requirement entirely. I've pushed an update to the current release without that requirement. I'll open a PR to have the package include the license in the source distribution in the future, but if it's OK I suggest we proceed with adding this now and wait for the package's next release to take care including the license file.

**Pull request merged**
